### PR TITLE
📖 Update outlline of 2023q1 PoC

### PR DESCRIPTION
**Pull request by MikeSpreitzer**
## Summary
This PR refreshes the 2023 PoC outline, primarily to remove the idea of using the TMC syncer.

## Related issue(s)

Fixes #

### CODE REVIEW

**Comment by MikeSpreitzer** [comment]
The force-push to 255f8789 is a rebase onto `main`.

File: docs/content/en/docs/Coding Milestones/PoC2023q1/outline.md [hunk 1]
```diff
@@ -171,16 +137,14 @@ future, and then those resources will join the previous category.
...
-modified version of TMC, which differs from today's TMC in ways
-summarized below in the [Syncers](#syncers) section.
+In very brief: the design approach is to achieve the multicast
+semantics of edge placement by two layers of activity.  Between the
+two layers sit _mailbox workspaces_: these exist in the center, and
+there is one for each edge cluster.  One layer of activity runs in the
+center and relates the edge placement problems to mailbox workspace
+contents.  The other layer is _syncers_, one in each edge cluster,
+that relate the corresponding mailbox contents with their local
+cluster.
```
Comment (waltforme) [inline docs/content/en/docs/Coding Milestones/PoC2023q1/outline.md:147]: Maybe this should be 'their local clusters' (plural)?

Comment (MikeSpreitzer) [reply docs/content/en/docs/Coding Milestones/PoC2023q1/outline.md]: yes, that's right, thanks.
Fixed in latest push

**Pull request merged**